packages/accounts-base/accounts_common.js

```diff
@@ -170,6 +170,18 @@ export class AccountsCommon {
       : null;
   }
 
+  /**
+   * @summary Get the current user record, or `null` if no user is logged in.
+   * @locus Anywhere
+   * @param {Object} [options]
+   * @param {MongoFieldSpecifier} options.fields Dictionary of fields to return or exclude.
+   */
+  async userAsync(options) {
+    const userId = this.userId();
```

Comment:
is this going to change? I think it will be sync as well?
```
    const userId = this.userId();
```

Reply:
No, userId is set on the DDP connection. So there is no need of making this async as it has on memory